c++/samples/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required (VERSION 2.8.12)
+
+project ("Cap'n'proto sample" CXX)
+
+# Needed to include generated Cap'n'proto headers
+set (CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Include CMake scripts
+include (FindPkgConfig)
+set (CMAKE_MODULE_PATH
+    "${PROJECT_SOURCE_DIR}/../cmake"
+    "${PROJECT_SOURCE_DIR}/cmake"
+)
+
+# Enable tests
+include(CTest)
+
+# Check for C++11 support
+include (C++11)
+CheckCXX11(FATAL_ERROR)
+
+# Add "-Wall" to compiler flags
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+
+# Find Cap'n'proto
+find_package (Capnp REQUIRED)
```

Comment:
It's some kind of CMake counterintuitivity – `find_package` checks if `Capnp_FOUND` is set, before it checked `CapnProto_FOUND`. In FindCapnp.cmake pkg_config searches for `capnp` and if it's found, it sets `Capnp_FOUND`, thus it's easier to rename just find_package module, rather than all variables and pkg_config file.